client/platform/desktop/backend/native/common.ts

```diff
@@ -801,6 +801,36 @@ async function findTrackandMetaFileinFolder(path: string) {
   return { trackFileAbsPath, metaFileAbsPath };
 }
 
+async function bulkMediaImport(path: string): Promise<DesktopMediaImportResponse[]> {
+  const children = await fs.readdir(path, { withFileTypes: true });
+  const results = await Promise.all(children.map(async (dirent) => {
+    let importResult: DesktopMediaImportResponse | undefined;
+
+    // Try to set import result, defaulting to undefined
+    const resolvedPath = npath.resolve(path, dirent.name);
+    try {
+      importResult = await beginMediaImport(resolvedPath);
+    } catch (e) {
+      // Do nothing, var will stay undefined
```

Comment:
It may be nice to put some sort of warning here for debugging purposes.  If I get an email with someone stating that bulk import isn't working this would help